skyportal/handlers/api/gcn.py

```diff
@@ -1211,8 +1211,19 @@ class Validator(Schema):
                     )
                     now_date = astropy.time.Time.now()
                     header_text.append(f"""DATE: {now_date}\n""")
+
+                    if (
+                        self.associated_user_object.affiliations is not None
+                        and len(self.associated_user_object.affiliations) > 0
+                    ):
+                        affiliations = ", ".join(
+                            self.associated_user_object.affiliations
+                        )
+                    else:
+                        affiliations = "Affiliation"
```

Comment:
It is a bit strange to have "Affiliation" as a placeholder. Would it be better to just cut off the "at..." and leave just the name?